python-tetris.py

```diff
@@ -1,111 +1,61 @@
 import pyglet
 
-window = pyglet.window.Window(500, 1000)
+UNIT = 40
+O_PIECE = [(4, 20), (5, 20), (4, 21), (5, 21)]
+I_PIECE = [(3, 20), (4, 20), (5, 20), (6, 20)]
+J_PIECE = [(3, 21), (3, 20), (4, 20), (5, 20)]
+L_PIECE = [(3, 20), (4, 20), (5, 20), (5, 21)]
+S_PIECE = [(3, 20), (4, 20), (4, 21), (5, 21)]
+Z_PIECE = [(3, 21), (4, 21), (4, 20), (5, 20)]
+T_PIECE = [(3, 20), (4, 20), (5, 20), (4, 21)]
+
+window = pyglet.window.Window(10*UNIT, 22*UNIT)
 # window.push_handlers(pyglet.window.event.WindowEventLogger())
 
-multiplier = 50
 global current_piece
 
 
-class square:
-    def __init__(self, x_orig, y_orig, side_length):
-        self.x_orig = x_orig
-        self.y_orig = y_orig
-        self.side_length = side_length
-
-    def origin(self):
-        return self.x_orig, self.y_orig
-
-    def position(self):
-        return (self.x_orig, self.y_orig,
-                self.x_orig+50, self.y_orig,
-                self.x_orig+50, self.y_orig+50,
-                self.x_orig, self.y_orig+50)
-
-    def move_right(self):
-        self.x_orig += 1*multiplier
+class piece:
+    def __init__(self, coords):
+        self.coords = []
+        for c in coords:
+            self.coords.append(tuple(x * UNIT for x in c))
+        self.size = len(coords)
+
+    def opengl_coords(self):
+        t = []
+        for c in self.coords:
+            t.append(
+                (c[0], c[1],
+                 c[0]+UNIT, c[1],
+                 c[0]+UNIT, c[1]+UNIT,
+                 c[0], c[1]+UNIT)
+            )
+        return t
 
     def move_left(self):
```

Comment:
Can you make all the move functions to one function and just accept a direction argument?